docs/reference/enumeration-classes.txt

```diff
@@ -0,0 +1,161 @@
+===================
+Enumeration Classes
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+MongoDB\\Model\\CollectionInfo
+------------------------------
+
+Definition
+~~~~~~~~~~
+
+.. phpclass:: MongoDB\\Model\\CollectionInfo
+
+   This class models information about a collection. Instances of this class are
+   returned by traversing a :phpclass:`MongoDB\\Model\\CollectionInfoIterator`,
+   which is returned by :phpmethod:`MongoDB\\Database::listCollections()`.
+
+Methods
+~~~~~~~
+
+.. toctree::
+   :titlesonly:
+
+   /reference/method/MongoDBModelCollectionInfo-getCappedMax
+   /reference/method/MongoDBModelCollectionInfo-getCappedSize
+   /reference/method/MongoDBModelCollectionInfo-getName
+   /reference/method/MongoDBModelCollectionInfo-getOptions
+   /reference/method/MongoDBModelCollectionInfo-isCapped
+
+----
```

Comment:
In the interest of keeping the left-hand TOC column concise, I wanted to bundle these insignificant classes into a single page. Without the horizontal lines, each class seemed to run right into the next, since we don't have large paragraphs to break things up like a tutorial page.

I'd love to discuss how we can improve the TOC for this later on. One oddity I noticed is that when you open one of these methods, the left-hand column will show methods for all classes that were on this page (which is likely because they all have the same TOC parent).

For now, I'd prefer to keep the horizontal lines in and clean it up if the CSS overhaul breaks something. HR tags are usually ignored, so it's low risk.